pontoon/settings/base.py

```diff
@@ -1127,14 +1130,23 @@ def account_username(user):
 except ValueError as e:
     raise ValueError(f"Error: {e}")
 
-# Used for Review Master and Translation Champion
+# Used for Translation Champion badge
 BADGES_TRANSLATION_THRESHOLDS = list(
     map(
         int,
         os.environ.get("BADGES_TRANSLATION_THRESHOLDS", "5, 50, 250, 1000").split(","),
     )
 )
-# Used for Community Builder
+# Used for Review Master badge
+# Thresholds are higher than Translation Champion because implicit rejections
+# on translations submissions and approvals also count against the Review Master badge.
```

Comment:
This is no longer the case? So we should use the same values as in BADGES_TRANSLATION_THRESHOLDS.
```suggestion
```